src/composables/useTransactions.ts

```diff
@@ -221,11 +225,40 @@ export default function useTransactions() {
   } = useWeb3();
   const { addNotification } = useNotifications();
   const { t } = useI18n();
+  const { fNum } = useNumbers();
 
   // COMPUTED
   const provider = computed(() => getWeb3Provider());
 
   // METHODS
+  function getSettledOrderSummary(
+    transaction: Transaction,
+    receipt: OrderReceipt
+  ) {
+    const details = transaction.details as GnosisTransactionDetails;
```

Comment:
@garethfuller this is the cool thing about keeping transaction details, it's possible to do all sorts of tweaks.